### ISSUE TITLE
Only import action creators when rule #4 is not applied?

### ISSUE THREAD

**Issue by fingermark** [erikras/ducks-modular-redux#19]
> ...and it will only import the action creators, ready to be passed to bindActionCreators().

Does this hold true even if rule 4 is applied?

> 4) MAY export its action types as UPPER_SNAKE_CASE, if an external reducer needs to listen for them, or if it is a published reusable library

**Comment by jcperez-ch** [comment]
@fingermark yes, there is a workaround that avoids to bindActionCreators() to  an `actionType` string instead of an actual action Creator function.

Nonetheless you can always import the action Creators separately as explained:

```js
import {loadWidgets, createWidget, updateWidget, removeWidget} from './ducks/widgets';
```

so you won't import `LOAD_WIDGETS` accidentally.

Another good approach is to wrap all your actionCreators in a namespaced exported variable

So in your duck file:
```js
actionCreators = {loadWidgets, createWidget, updateWidget, removeWidget};
exports {actionCreators}
```

And in your component:
```js
import {actionCreators} from './ducks/widgets';
bindActionCreators({...actionCreators}, dispatch);
```

If that ever makes sense... it worked for me at least.
